pytorch_lightning/trainer/data_loading.py

```diff
@@ -26,6 +26,13 @@
 else:
     XLA_AVAILABLE = True
 
+try:
+    import horovod.torch as hvd
+except ImportError:
+    HOROVOD_AVAILABLE = False
+else:
+    HOROVOD_AVAILABLE = True
```

Comment:
should we throw a warning or error if the user requests horovod but doesn't have it installed?
in which case the warning should give the install instructions?

Reply:
Good idea.  I added a check for this, let me know what you think.